project/dependencies.scala

```diff
@@ -1,7 +1,7 @@
 import sbt._
 
 object Dependencies {
-  val capiVersion = "19.2.3"
+  val capiVersion = "20.0.2-PREVIEW.update-to-latest-version-of-content-api-models-scala.2024-01-25T1416.253f60e5"
```

Comment:
This is the preview of https://github.com/guardian/content-api-scala-client/pull/405